### ISSUE TITLE
Warning: React.createElement: type is invalid

### ISSUE THREAD

**Issue by sofven** [dialectlabs/react#42]
Hi. We can't seem to make the react ui component work. We're getting this error:

![Screen Shot 2022-03-25 at 12 40 34 PM](https://user-images.githubusercontent.com/87349746/160055515-62996a0f-a72e-4216-ae1d-446e1d74b3c3.png)

```
function WrappedNotificationsButton(props) {
  const wrapperRef = (0, react_1.useRef)(null);  <--- The first link points to this line 
  const bellRef = (0, react_1.useRef)(null);
  const [open, setOpen] = (0, react_1.useState)(false);
  ...
```

We're using these versions:
```
"dependencies": {
    "@dialectlabs/react": "^0.1.7",
    "@dialectlabs/react-ui": "^0.1.16",
    ...
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-scripts": "^5.0.0",
    ...
}
```

Thank you.

**Comment by fsher** [comment]
Hey there @sofven! Thanks for creating this issue. Actually we've noticed it as well recently and currently working on a fix.

A couple of follow-up questions. I'm guessing you are using CRA with webpack 5, is that correct?

Here is a PR with a potential fix that should resolve this issue - https://github.com/dialectlabs/react/pull/41

**Comment by sofven** [comment]
Hi. Thank you for the response. Yes, we're using CRA 5. Will check this, thanks!

**Comment by fsher** [comment]
@sofven Hey! We've released react-ui@0.2.0 that has this fixed. Please try it out. :)

Let me know if it works for you. Thanks!

**Comment by sofven** [comment]
Hi. Yes, it works now after being updated to 0.2.0! Thank you!
